packages/cli-helpers/src/auth/authTasks.ts

```diff
@@ -154,7 +154,7 @@ const addAuthImportToRoutes = (content: string) => {
   const contentLines = content.split('\n')
   // Find the last import line that's not a .css or .scss import
   const importIndex = contentLines.findLastIndex((line: string) =>
-    /^\s*import (?!.*(?:.css'|.scss'))/.test(line),
+    /^\s*import (?!.*(?:.css['"]|.scss['"]))/.test(line),
```

Comment:
You could probably write a regex that makes sure the opening and closing quotes are the same - as they would be. I don't think we need to be that surgical here and now.

Reply:
We could. And we don't have to. But if we wanted it'd be very easy 😉  Just create a capture group and then back-reference it 🙂

Reply:
`/(b)\1/i.test("bB"); // true`

Reply:
Hahah if you want to push up a commit for it then happy for you too 😉 but otherwise I'm not going to 😆